website/content/v1.4/introduction/quickstart.md

```diff
@@ -12,10 +14,10 @@ The easiest way to try Talos is by using the CLI (`talosctl`) to create a cluste
 
 #### `talosctl`
 
-Download `talosctl`:
+Download `talosctl` (macOS or Linux):
 
 ```bash
-curl -sL https://talos.dev/install | sh
+brew install siderolabs/tap/talosctl
```

Comment:
should we  keep the curl command for non-brew users, i'm not sure how other tools document stuff, only document the brew method?